example/src/components/Counter/index.tsx

```diff
@@ -20,7 +20,7 @@ export interface CounterProps {
 // };
 
 export function Counter(props: CounterProps) {
-  const counter = useBean(CounterModel, { props });
+  const counter = useBean(() => new CounterModel(), { props });
```

Comment:
Any difference?

Reply:
a factory method to create the bean for passing some object from react context into stated bean

Reply:
I still can't get the point, is the previous code not working as expected as current one now? I supposed they were just exactly same.